README.rst

```diff
@@ -359,4 +359,4 @@ Documentation for Underlying Libraries
 
 * PyQuery (pdf.pq): pyquery_documentation_.
 
-.. _pyquery_documentation: http://packages.python.org/pyquery/
```

Comment:
I didn't touch this line...I have no idea what's different about it. I suspect Github stripped the last line return?